venia-integration-tests/src/plugins/resizeSnapshotPlugin.js

```diff
@@ -0,0 +1,65 @@
+const jimp = require('jimp');
+const fs = require('fs');
+const { matchImageSnapshotPlugin } = require('cypress-image-snapshot/plugin');
+
+const resizeSnapshot = imageConfig => {
```

Comment:
Is this the function that resolves differences in terms of resolution and pixel density?

Reply:
Yes, that's the one.